uui-editor/src/plugins/videoPlugin/AddVideoModal.tsx

```diff
@@ -64,7 +64,15 @@ export function AddVideoModal({ editor, success, abort, ...props }: AddVideoModa
                 <ModalHeader title="Add video" onClose={ abort } />
                 <FlexRow cx={ css.inputWrapper }>
                     <LabeledInput label="Video url">
-                        <TextInput value={ src } onValueChange={ setSrc } autoFocus />
+                        <TextInput
+                            value={ src }
+                            onValueChange={ (v) => {
+                                if (v) {
```

Comment:
Why do we need such condition here? I think it will broke set of empty value to the input, in case when you have some value and then decided to remove it.

Reply:
Changed